src/utils/geometry/delta-apply.ts

```diff
@@ -242,7 +242,7 @@ export function applyTreeDeltas(
         applyBoxDelta(box, delta)
 
         // If this is a draggable ancestor, also incorporate the node's transform to the layout box
-        if (!isRelative && node.getProps().drag) {
+        if (node.getProps().drag) {
```

Comment:
This was only applying drag transforms to children that were being projected absolutely. So children were jumping as they weren't being transformed in the same way.